data/json/items/gun/flammable.json

```
@@ -5,19 +5,20 @@
     "looks_like": "m79",
     "type": "GUN",
     "name": { "str": "flamethrower" },
-    "description": "A large flamethrower with substantial gas reserves.  Very menacing and deadly.",
+    "description": "A menacing and deadly weapon in the right hands, this is a large back-mounted flamethrower with sufficient room for substantial gas reserves.  Before the Cataclysm, flamethrowers like this one were primarily used to start agricultural burns and clear dead vegetation.  However, when the dead rose and bullets weren’t enough to put them back in their graves, numbers of flamethrowers were procured as last-ditch emergency weapons by the US government.",
```

```suggestion
    "description": "A menacing and deadly weapon in the right hands, this is a large back-mounted flamethrower with sufficient room for substantial gas reserves.  Before the Cataclysm, flamethrowers like this one were primarily used to start agricultural burns and clear dead vegetation.  However, when the dead rose the US military tried to use them to push back against the undead hordes.",
```
Since they don't really spawn more aside from zombie burner we should probably not mention any kind of increased production